core/psh/Makefile

```
@@ -1,10 +1,16 @@
 #
 # Makefile for Phoenix-RTOS psh (Phoenix SHell)
 #
-# Copyright 2018, 2019, 2020 Phoenix Systems
+# Copyright 2018, 2019, 2020, 2021 Phoenix Systems
 #
 
-$(PREFIX_PROG)psh: $(addprefix $(PREFIX_O)core/psh/, cat.o fs.o kill.o ls.o mem.o perf.o ps.o psh.o reboot.o top.o)
+PSH_COMMANDS ?= bind cat exec help kill ls mem mkdir mount perf ps reboot runfile sync sysexec top touch
+OBJS = bind cat exec help kill ls mem mkdir mount perf ps reboot runfile sync sysexec top touch
+OBJ_FEATURED=$(addsuffix -empty.o,$(filter-out $(PSH_COMMANDS),$(OBJS))) $(addsuffix .o,$(filter $(PSH_COMMANDS),$(OBJS)))
```

nitpicking, spaces aroung the equal sign:
```suggestion
OBJ_FEATURED = $(addsuffix -empty.o,$(filter-out $(PSH_COMMANDS),$(OBJS))) $(addsuffix .o,$(filter $(PSH_COMMANDS),$(OBJS)))
```
Personally, I prefer using spaces after each comma sign, but I see that in some complex Makefile expressions we don't use them, so you could leave it as it is.